src/datasetsite-csp.mustache

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE HTML>
+<!--
+  OpenActive Dataset Site Template version 1.0.16, from https://unpkg.com/@openactive/dataset-site-template@1.0.16/dist/datasetsite-csp.mustache
+  
+  This HTML file must reference a self-hosted 'datasetsite.styles.v1.0.16.css' file, co-located with the rest
+  of the static assets from the following archive:
+  https://unpkg.com/@openactive/dataset-site-template@1.0.16/dist/datasetsite-csp.static.zip
+-->
+<!--
+  Design: Identity by HTML5 UP
+  html5up.net | @ajlkn
+  Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+
+  DCAT Template: Nick Evans / imin
+  imin.co | @nickevans / @_imin_
+  For openactive.io
+  Free for personal and commercial use under the CC-BY v4.0 license (https://creativecommons.org/licenses/by/4.0/)
+-->
+<html prefix="dct: http://purl.org/dc/terms/
+              rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+              dcat: http://www.w3.org/ns/dcat#
+              foaf: http://xmlns.com/foaf/0.1/
+              og: http://ogp.me/ns#
+              odrs: http://schema.theodi.org/odrs#"
+      lang="en">
+<head>
+    <title>{{name}} - Open Data</title>
+
+    <meta name="title" content="{{name}}  - Open Data" />
+    <meta name="identifier" content="{{url}}" />
+    <meta name="Keywords" content="{{#keywords}}{{.}},{{/keywords}}Open Data" />
+    <meta name="Description" content="{{description}}" />
+    <meta name="language" content="English" />
+
+    <meta property="og:title" content="{{name}} - Open Data" />
+    <meta property="og:description" content="{{description}}" />
+    <meta property="og:locale" content="en_GB" />
+    <meta property="og:url" content="{{url}}" />
+    <meta property="og:image" content="{{publisher.logo.url}}" />
+
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    
+    <!-- JSON-LD conforming to Google Structured Data specification. -->
+    <script type="application/ld+json">
+{{{jsonld}}}
+    </script>
+    
+    <!--
+      This stylesheet href must reference a self-hosted 'datasetsite.styles.v1.0.16.css' file in
+      the same directory as the other static assets sourced from the following archive:
+      https://unpkg.com/@openactive/dataset-site-template@1.0.16/dist/datasetsite-csp.static.zip
+    -->
+    <link rel="stylesheet" href="{{{staticAssetsPathUrl}}}/datasetsite.styles.v1.0.16.css" crossorigin="anonymous">
+
+</head>
+<body id="booking" background="{{backgroundImage.url}}">
```

Comment:
If this URL has any special chars in it then it'll get HTML encoded here, might be better to use triple brackets as above